### ISSUE TITLE
Video Button

### ISSUE THREAD

**Issue by kainekh** [friendica/friendica#9910]
I do not know if this is an issue where I set it up wrong or need to check a simple setting; I've made a horde of mistakes setting this up. I apologize if this is something stupid simple:

When we try to upload a video, there is no "upload" button like there is on the photo.

I'm a complete noob at this, so I'm getting my feet wet by doing. Are there any specific log or .ini files I should post?

**Comment by MrPetovan** [comment]
Hi, Friendica doesn’t currently handle video upload if it ever did. The Videos section is misleading and probably should he removed.

I’d suggest you to use PeerTube to post videos and reshare from your Friendica account since both support ActivityPub.

**Comment by kainekh** [comment]
OK, thank you for the response :)

**Comment by MrPetovan** [comment]
You're welcome, sorry about the confusion.

**Comment by AlfredSK** [comment]
Hmm. I'm pretty sure Friendica can handle videos as attachments. The attached video should appear in the video section after the post has been submitted.

At least that was the case in the past. I didn't try it lately. And yes, the whole video stuff is complicate and confusing.

**Comment by AlfredSK** [comment]
Yep. It's still working. But the workflow is for Friendica experts only. ;-)

![image](https://user-images.githubusercontent.com/3656054/107143893-cee27f00-6937-11eb-8082-85cd43d7c5e1.png)

![image](https://user-images.githubusercontent.com/3656054/107143906-efaad480-6937-11eb-9d15-2c8ceb31ebe5.png)

**Comment by SpcCw** [comment]
@AlfredSK - yes, it works but only with relatively small files. Even when upload limit is large uploading files of substantial size often fails.

Could be handy to share short mp4 clips which are used as GIFs these days but not real videos :)

**Comment by mikauskas** [comment]
> Hi, Friendica doesn’t currently handle video upload if it ever did. The Videos section is misleading and probably should he removed.
> 
> I’d suggest you to use PeerTube to post videos and reshare from your Friendica account since both support ActivityPub.

Don't you dare thinking about removing video function! Friendica has to be better, not worse than Facebook.
When issue with not showing even Peertube videos will be fixed on the latest versions? 
https://github.com/friendica/friendica/issues/9666

**Comment by MrPetovan** [comment]
None of the current core developers know how to handle video properly, and the current UI is confusing, hence my suggestion. I do believe that having a misleading Videos section with a different behavior than the Photos section is worse for Friendica usability than not having it at all.

**Comment by SpcCw** [comment]
> I do believe that having a misleading Videos section with a different behavior than the Photos section is worse for Friendica usability than not having it at all.

How about renaming it to something like "Media" and using it for listing not just videos but any attached files (which now are listed only in the media browser when creating new post)? It could be useful for other things like publishing one's PGP key for example.

At the same time I would keep current video upload functionality as it is because it is perfectly usable for short clips and longer, problematic videos won't get uploaded anyway because of the server upload limit.

**Comment by mikauskas** [comment]
> @AlfredSK - yes, it works but only with relatively small files. Even when upload limit is large uploading files of substantial size often fails.
> 
> Could be handy to share short mp4 clips which are used as GIFs these days but not real videos :)

While digging for other things noticed settings in PHP options might help with this:
![Screenshot_2021-02-08 cPanel -(2)](https://user-images.githubusercontent.com/76782677/107239980-4f30df00-6a21-11eb-8370-1bdefc6d5075.png)
